molecule/default/destroy.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Destroy
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
+  vars:
+    molecule_file: "{{ lookup('env', 'MOLECULE_FILE') }}"
+    molecule_instance_config: "{{ lookup('env',' MOLECULE_INSTANCE_CONFIG') }}"
+    molecule_yml: "{{ lookup('file', molecule_file) | molecule_from_yaml }}"
+  tasks:
+    - name: Destroy molecule instance(s)
+      molecule_vagrant:
+        instance_name: "{{ item.name }}"
+        platform_box: "{{ item.box }}"
+        provider_name: "{{ molecule_yml.driver.provider.name }}"
+        force_stop: "{{ item.force_stop | default(True) }}"
+
+        state: destroy
+      register: server
+      with_items: "{{ molecule_yml.platforms }}"
+
+    # Mandatory configuration for Molecule to function.
+
+    - name: Populate instance config
```

Comment:
what - i dont understand why this bottom section is needed. the instance config should get recreated depending on state (from `molecule/securedrop/.molecule/state.yml`). At least that was my understanding. Oh maybe this is from upstream molecule? nm -- guess is this first time i've seen a vagrant template for molecule 2.x . Now im just talking out loud here.... so..... how about them local sports team!?

Reply:
Yes, that's from the boilerplate Molecule v2 from running `molecule init role -r <role_name> --driver-name vagrant`.